plugins/chat/assets/javascripts/discourse/templates/preferences/chat.hbs

```diff
@@ -1,6 +1,6 @@
 <label class="control-label">{{i18n "chat.title_capitalized"}}</label>
 
-<div class="control-group chat-setting">
+<div class="control-group chat-setting user_chat_enabled">
```

Comment:
Since these don't follow BEM for CSS which we are trying to do now, and these are more of a data thing, these should be in data attributes e.g. `data-chat-setting="user_chat_enabled"`, `data-chat-setting="user_chat_ignore_channel_wide_mention"`

Reply:
Done!